cms/static/js/views/paging_header.js

```diff
@@ -29,7 +30,7 @@ define([
                     messageHtml = this.messageHtml(),
                     isNextDisabled = lastPage === 0 || currentPage === lastPage;
 
-                this.$el.html(_.template(pagingHeaderTemplate)({ messageHtml: messageHtml}));
+                this.$el.html(_.template(pagingHeaderTemplate)({ messageHtml: messageHtml, HtmlUtils: HtmlUtils}));
```

Comment:
You don't need to inject ``HtmlUtils`` if you use ``HtmlUtils.template`` (it injects both this and ``StringUtils``).